packages/result/src/lib/Parsers.ts

```diff
@@ -0,0 +1,34 @@
+import { err, ok, Result } from './Result';
+
+type Awaitable<T> = PromiseLike<T> | T;
+
+/**
+ * Creates a {@link Result} out of a callback.
+ * @typeparam T The result's type.
+ * @typeparam E The error's type.
+ */
+export function from<T, E = unknown>(cb: (...args: unknown[]) => T): Result<T, E> {
+	try {
+		return ok(cb());
+	} catch (error) {
+		return err(error as E);
+	}
+}
+
+// eslint-disable-next-line @typescript-eslint/ban-types
+function isFunction(input: unknown): input is Function {
```

Comment:
`function` expressions get lifted up by JS so the order of definition doesn't matter. You can place this one at the bottom so the exports are above it.